pkg/component/shared/istio.go

```diff
@@ -91,6 +91,7 @@ func NewIstio(
 
 	policyLabels := commonIstioIngressNetworkPolicyLabels(vpnEnabled)
 	policyLabels[toKubeAPIServerPolicyLabel] = v1beta1constants.LabelNetworkPolicyAllowed
+	policyLabels[v1beta1constants.LabelNetworkPolicyToRuntimeAPIServer] = v1beta1constants.LabelNetworkPolicyAllowed
```

Comment:
Maybe add a comment why this is needed? Might not be immediately obvious :)

Reply:
Good point. Added a comment in https://github.com/gardener/gardener/pull/9300/commits/be314a61d048f5cc7c58a149ef3128568015855c.
WDYT?